types/recharts/index.d.ts

```diff
@@ -634,6 +635,7 @@ export interface ReferenceAreaProps extends Partial<PresentationAttributes> {
     yAxis?: object;
     isFront?: boolean;
     alwaysShow?: boolean;
+    ifOverflow?: 'hidden' | 'visible' | 'discard' | 'extendDomain';
```

Comment:
I can agree, but I couldn't figure out when in the file the choice were to go with named types and when not to, so I thought you might have taken some info from the recharts, and when I looked there, I couldn't easily say that this would be the same on all the types "forever" or just temporary.

If you want to, I can update the PR.

Reply:
I would expect that the three kinds of reference areas would support the same overflow options.

Reply:
Even if they diverge in the future, extracting the type now will make it easier to determine which consumers need to be revisited.